core/jazz_ui/src/app/secondary-components/filter-modal/filter-modal.component.ts

```diff
@@ -1,25 +1,47 @@
-import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {Component, EventEmitter, Input, OnInit, Output,OnChanges, SimpleChanges, SimpleChange} from '@angular/core';
```

Comment:
remove onchanges, simplechanges if not used

Reply:
removed